bom/platform-next/pom.xml

```diff
@@ -0,0 +1,1206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.hibernate.search</groupId>
+        <artifactId>hibernate-search-parent</artifactId>
+        <version>8.1.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>hibernate-search-platform-next-bom</artifactId>
+    <name>Hibernate Search - Platform Lucene 10 BOM</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <!-- This is a publicly distributed module that should be published: -->
+        <deploy.skip>false</deploy.skip>
+
+        <!-- These versions will be checked against the ones resolved by Maven for the project in the enforcer rule -->
+        <version.bom.org.hibernate.orm>7.0.7.Final</version.bom.org.hibernate.orm>
+        <version.bom.org.apache.lucene>10.2.2</version.bom.org.apache.lucene>
+        <version.bom.org.elasticsearch.client>9.0.4</version.bom.org.elasticsearch.client>
+        <version.bom.software.amazon.awssdk>2.31.2</version.bom.software.amazon.awssdk>
+        <version.bom.io.smallrye>3.3.0</version.bom.io.smallrye>
+        <version.bom.org.jboss.logging.processor>3.0.4.Final</version.bom.org.jboss.logging.processor>
+        <version.bom.org.jboss.logging>3.6.1.Final</version.bom.org.jboss.logging>
+        <version.bom.org.hibernate.models>1.0.0</version.bom.org.hibernate.models>
+        <version.bom.org.apache.avro>1.12.0</version.bom.org.apache.avro>
+        <version.bom.com.carrotsearch>0.10.0</version.bom.com.carrotsearch>
+        <version.bom.com.google.code.gson>2.13.1</version.bom.com.google.code.gson>
+        <version.bom.jakarta.batch>2.1.1</version.bom.jakarta.batch>
+        <version.bom.jakarta.enterprise>4.0.1</version.bom.jakarta.enterprise>
+        <version.bom.jakarta.inject>2.0.1</version.bom.jakarta.inject>
+        <version.bom.jakarta.persistence>3.2.0</version.bom.jakarta.persistence>
+        <version.bom.jakarta.transaction>2.0.1</version.bom.jakarta.transaction>
+        <version.bom.jakarta.xml.bind>4.0.2</version.bom.jakarta.xml.bind>
+        <version.bom.org.jberet>3.1.0.Final</version.bom.org.jberet>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!--
+                Hibernate Search artifacts listed explicitly as we cannot exclude some from the bom import,
+                and we want to skip the lucene-next ones for this pom to not mix different Lucene versions
+            -->
```

Comment:
Makes me think you probably need a `hibernate-search-next-bom` too?

Reply:
This one seems to work unexpectedly ok 😃 
Since it does not manage Lucene directly each backend transitevely brings the version it wants as long as you do not use two lucene backends at the same time 😃